lib/nx_zlib.h

```diff
@@ -123,7 +123,7 @@ struct nx_config_t {
 	uint32_t per_job_len;          /* less than suspend limit */
 	uint32_t strm_def_bufsz;
 	uint32_t soft_copy_threshold;  /* choose memcpy or hwcopy */
-	uint32_t compress_threshold;   /* collect as much input */
+	uint32_t cache_threshold; /** Cache input before processing */
```

Comment:
There are two '*' instead of one on the comment.

Reply:
This is intentional.  That means it will appear in the Doxygen documentation.